platform/linux-dpdk/include/odp/api/plat/version_types.h

```diff
@@ -0,0 +1 @@
+../../../../../linux-generic/include/odp/api/plat/version_types.h
```

Comment:
Work that needs to be done needs to be done. Deferring it only invites not doing it at all as other "higher priority" items get pushed ahead of it. The result that an inflexible structure gets codified, leading to long-term development / maintenance headaches. That was the essence of the discussions we've had over the past few weeks: It's better to do things right than quickly, which is why we punted on a formal July milestone. Now we just have to follow-through on that decision.

The attraction of the odp-dpdk-import branch is that it allows this work to occur in parallel. Work that needs to be done / experimented with sooner can be done directly on that branch while the more methodical restructuring work needed to realize the cloud-dev vision can proceed independently. In the meantime, we're no longer bottlenecked by one person working on odp-dpdk as everyone can start contributing to this restructuring effort since the source material is now staged.